app/controllers/books_controller.rb

```diff
@@ -17,6 +17,7 @@ def new
   end
 
   def update
+    authorize @book, policy_class: BookPolicy
```

Comment:
It can refer to `BookPolicy` from the controller name.

```suggestion
    authorize @book
```